### DISCUSSION TITLE
Blog Post: Deep Dive into Reverse Engineer

### DISCUSSION THREAD

**Discussion by altugbakan** [stellar/sorobanathon#82] [General]
Hello!

I have created a blog post about tackling the reverse engineer quest without even leaving your terminal! Since reverse engineer quest is not one of the usual quests so far, the blog post includes some Linux tools to handle the quest. Check it out [here](https://dev.to/altug/soroban-quest-reverse-engineer-a95)!

You can find my other posts about other quests on [my dev.to blog](https://dev.to/altug).

**Comment by achallagundla** [comment]
Heya @altugbakan  ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1500 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

PS: No need to submit the form if you already have, just update me on discord if your wallet address has changed.
